gh/index.js

```diff
@@ -35,13 +35,19 @@ module.exports = yeoman.generators.Base.extend({
         name: 'branch',
         message: 'What branch would you like to deploy from?',
         default: 'master'
-      }
+      },
+      {
+        name: 'connectionType',
+        message: 'Use HTTPS or SSL?',
```

Comment:
Would you mind changing this to:

'Use HTTPS or SSL to connect to GitHub?'

Just to help first time users